.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,5 +37,3 @@ request, mention that information here.-->
 - [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
 - [ ] Variables are documented in the README.md
 - [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
-
-:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.
```

Comment:
I guess in the past this comment was targeted at bitnami folk who may have attempted a PR against this chart... where as now bitnami folk *should* update this chart. But I'm not sure what the best generally. We'll be making changes in our development chart and then syncing them here via a PR for development-related changes. Perhaps we could still say:

:warning: Updates and bug fixes for the chart belong in this repository but functionality for new Kubeapps features should be included in the [development chart in the Kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps).

Maybe it's not necessary, see what you think.

Reply:
Nice suggestion! What do you think @carrodher ?

Reply:
Since the development workflow will continue to be in the other repository, it makes sense, so we avoid possible divergences

Reply:
I'm ok with either option; no strong opinion here.

However, since our main focus is to consider Kubeapps as a completely separate thing, perhaps adding this disclaimer in every pull request for the whole bitnami chart catalog is too muuch.
We can have it documented internally if necessary, but I'd rather avoid adding Kubeapps-related information. in the bitnami repo.

Reply:
🤷‍♂️  no strong opinion here

Reply:
> However, since our main focus is to consider Kubeapps as a completely separate thing, perhaps adding this disclaimer in every pull request for the whole bitnami chart catalog is too muuch.
> We can have it documented internally if necessary, but I'd rather avoid adding Kubeapps-related information. in the bitnami repo.

+1, I'd not taken into account that this was a standard disclaimer for all PRs to the bitnami chart catalog. Let's just document it internally.